agent/lib/kontena/workers/stats_worker.rb

```diff
@@ -117,7 +118,7 @@ def send_container_stats(container)
         }
       }
       self.queue << event
-      send_statsd_metrics(container.dig(slice_key, :aliases, 0), event[:data])
+      send_statsd_metrics(container.dig(:aliases, 0), event[:data])
```

Comment:
Is this assumption of `"aliases": [name, id]` sensible?

It looks like the current cAdvisor implementation always returns these in this order: https://github.com/google/cadvisor/blob/v0.24.1/container/docker/handler.go#L222

But it's certainly not part of the API...